clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -117,9 +117,9 @@ describe("System management with Plus features", () => {
           fixture: "taxonomy/custom-metadata/custom-field/list.json",
         }
       ).as("getCustomFields");
-      cy.intercept("PUT", `/api/v1/plus/custom-metadata/custom-field`, {
-        fixture: "taxonomy/custom-metadata/custom-field/update-party.json",
-      }).as("updateParty");
+      cy.intercept("POST", `/api/v1/plus/custom-metadata/custom-field/bulk`, {
+        body: {},
+      }).as("bulkUpdateCustomField");
```

Comment:
this is a test that needed to be updated after the changes in https://github.com/ethyca/fides/pull/4294/commits/ba7f66bcba991fa768e36d9f7120771598544fdd